### PR TITLE
Added translation files for Brazilian Portuguese

### PR DESCRIPTION
**Pull request by Patola**
As per directions given on PR #676, I created the translation files for brazilian portuguese. All strings are translated and have been subjected to the latest version of template file using msgmerge from gettext.

### CODE REVIEW

File: resources/i18n/ptbr/fdmprinter.def.json.po [hunk 1]
```diff
@@ -0,0 +1,5178 @@
...
+msgctxt "speed_travel label"
+msgid "Travel Speed"
+msgstr "Velocidade de Viagem"
+
+#: fdmprinter.def.json
+msgctxt "speed_travel description"
+msgid "The speed at which travel moves are made."
+msgstr ""
+"Velocidade em que ocorrem os movimentos de viagem (movimentação do extrusor "
+"s
```
Comment (Ghostkeeper) [inline resources/i18n/ptbr/fdmprinter.def.json.po]: Hmm, I think Github isn't happy with these long files. This comment was meant to be placed at the translation of "Anti Overhang Mesh".

**Comment by Patola** [comment]
Thans for the comments! Yes, I think stripping all quotes from 'Jerk' is better. It is just that it is already used as a jargon term, so translating to a portuguese word would confuse people instead of clarifying. For 'anti support mesh', the reason was that the translation of 'overhang' to portuguese is an awkward expression - "seções pendentes" - and using "anti seções pendentes" would make it even more awkward. But awkward does not equal obscure, it would be easily understood this way also. Should I edit my PR to make these changes?

**Comment by Ghostkeeper** [comment]
We found that the plural forms were wrong, by the way. I corrected them on master.

**Comment by Ghostkeeper** [comment]
Anti support mesh is fine as it is then.

If you feel like editing the "Jerk" thing, feel free. I've already merged this pull request, so please make another one if you feel like making the modification.

**Comment by Patola** [comment]
I came with a more appropriate translation for anti-overhang so I got rid of 'anti-suporte'. Also, I made the changes to jerk with quotes. PR#1386

**Pull request merged**